dummy-ignition/main.go

```diff
@@ -1,10 +1,8 @@
 package main
```

Comment:
Can you add a file documentation why it is needed?

Reply:
Done.